docs/docs/code/nodejs/README.md

```diff
@@ -211,33 +341,45 @@ Sometimes you want to end your workflow early, or otherwise stop or cancel the e
 - You only want to run your workflow for users in the United States. If you receive a request from outside the U.S., you don't want the rest of the code in your workflow to run.
 - You may use the `user_id` contained in the event to look up information in an external API. If you can't find data in the API tied to that user, you don't want to proceed.
 
-**In any code step, calling the `$end()` function will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
+**In any code step, calling the `$.end()` function will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
```

Comment:
Need to modify this section to use `$.flow.exit` instead of `$.end`

Reply:
Are there other `$.flow` options and do they need documented?

Reply:
https://pipedream.com/docs-v2/workflows/events/#event-format